src/ontology/mondo-edit.obo

```diff
@@ -376435,7 +376399,10 @@ xref: ICD10CM:Q39.6 {source="Orphanet:91358/attributed", source="Orphanet:91358/
 xref: ICD9:750.4 {source="MONDO:relatedTo", source="MONDO:i2s"}
 xref: Orphanet:91358 {source="MONDO:equivalentTo"}
 xref: SCTID:204667006 {source="MONDO:equivalentTo"}
-is_a: MONDO:0015207 {source="Orphanet:91358"} ! non-syndromic esophageal malformation
+is_a: MONDO:0020019 ! digestive tract malformation
+is_a: MONDO:0021147 ! disorder of development or morphogenesis
+relationship: disease_has_location UBERON:0001043 ! esophagus
+relationship: has_characteristic MONDO:0021136 ! rare
```

Comment:
@nicolevasilevsky did you add this manually?

Reply:
or maybe that is the re-wiring?

Reply:
this was rewiring, I didn't add any of these